platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -92,6 +92,18 @@ Starting with Gradle 9.0, the Gradle API is annotated using JSpecify instead.
 
 For more details and potential breakages, see the dedicated [upgrading guide section](userguide/upgrading_version_8.html).
 
+#### Introduced `RootComponentIdentifier`
+
+A new subtype of `ComponentIdentifier` has been introduced which identifies the root component of a dependency graph. 
+When a configuration is resolved, it is transformed into a synthetic variant that serves as the root of a dependency graph.
+The root component exists solely to own the root variant.
+
+Dependency graphs resolved from `buildscript` Configurations and detached Configurations will have a component identified by a `RootComponentIdentifier` at the root of their graph.
```

Comment:
```suggestion
Dependency graphs resolved from `buildscript` configurations and detached configurations will have a component identified by a `RootComponentIdentifier` at the root of their graph.
```

Reply:
Is there javadoc and user manual docs we could link to?

Reply:
I couldn't find any userguide documentation on buildscript blocks.

We could link to the `RootComponentIdentifier` javadoc, but I didn't find any other examples of linking to javadocs in the release notes, so I am not sure how